src/pages/RedeemCode.tsx

```
@@ -47,49 +44,43 @@ const RedeemCode: React.FC = () => {
-                            color: "white",
-                        },
-                        "& .MuiInputLabel-root.Mui-focused": {
-                            color: "lightblue",
-                        },
-                        input: { color: "white" },
-                    }}
-                />
+            <div className="overlay" />
+            <h1 className="title">Godbless</h1>
+            <div className="form-container"
+                onMouseEnter={(e) => e.currentTarget.style.transform = "scale(1.05)"}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)